packages/@webex/plugin-meetings/src/recording-controller/index.ts

```diff
@@ -78,36 +76,127 @@ export default class RecordingController {
    * @private
    * @memberof RecordingController
    */
-  private initialize(request: MeetingRequest, info: LocusInfo) {
+  private initialize(request: MeetingRequest) {
     this.request = request;
-    this.info = info;
-    this.extract(info);
   }
 
   /**
-   * @param {LocusInfo} info
+   * @param {Object} options
    * @returns {void}
    * @public
    * @memberof RecordingController
    */
-  public set(info: LocusInfo) {
-    this.extract(info);
+  public set(options: {
```

Comment:
is this function called from anywhere now? it seems to be not used

Reply:
it's not used, except for in the tests and just internally calls other functions, however, I think with your question above, I could use it to make the code simpler in meeting/index and solve the "hardening" issue with timing of events.

Reply:
yeah, let's do that